torch/csrc/autograd/python_variable.cpp

```diff
@@ -291,7 +291,7 @@ int THPVariable_set_volatile(THPVariable *self, PyObject *obj)
 PyObject *THPVariable_get_output_nr(THPVariable *self)
 {
   auto& var = *self->cdata;
-  return PyInt_FromLong(var.output_nr);
+  return PyLong_FromLong(var.output_nr);
```

Comment:
Reverted a8857be5b5ff5cf73de04b8821248aef9ae6b703 and f34595362cce809f15da1e8e558153e3d946835b. Included `THP.h` in fe8b1524cc4fdd051ddecb7172849b1a28ba0ff0.